view/script/trial.js

```diff
@@ -1,10 +1,42 @@
+'use strict'
+
+function PlotDoughNut(data)
```

Comment:
it looks like you are manually drawing the doughnut chart.
impressive, but please use the default draw functionality provided by chart.js.

https://nnnick.github.io/Chart.js/docs-v2/#doughnut-pie-chart-example-usage